internal/database/conf.go

```diff
@@ -39,6 +39,18 @@ type ConfStore interface {
 	// responsible for ensuring this or that the response never makes it to a user.
 	SiteGetLatest(ctx context.Context) (*SiteConfig, error)
 
+	// ListSiteConfig will list the configs of type "site".
+	//
+	// 🚨 SECURITY: This method does NOT verify the user is an admin. The caller is
+	// responsible for ensuring this or that the response never makes it to a user.
+	ListSiteConfig(context.Context, SiteConfigListOptions) ([]*SiteConfig, error)
```

Comment:
I know it sounds dumb, but I think `List*` methods should have plural in them 😄 

```suggestion
	ListSiteConfigs(context.Context, SiteConfigListOptions) ([]*SiteConfig, error)
```